vms/platformvm/service.go

```diff
@@ -2191,11 +2181,7 @@ type GetTotalStakeReply struct {
 
 // GetTotalStake returns the total amount staked on the Primary Network
 func (service *Service) GetTotalStake(_ *http.Request, _ *struct{}, reply *GetTotalStakeReply) error {
-	validators, ok := service.vm.Validators.GetValidators()
-	if !ok {
-		return errNoPrimaryValidators
-	}
-	reply.Stake = json.Uint64(validators.Weight())
+	reply.Stake = json.Uint64(service.vm.Validators.Weight())
```

Comment:
The extra checks which was done before isn't happening now. Is it because the `service.vm.Validators` always be non-nil in all these calls?

Reply:
That check was only there because we previously (in an earlier version that what you see on the right) got the validators by chain ID. I left it there to keep the changes minimally invasive, but now it just doesn't make sense anymore.